src/app/components/Byline/utilities/bylineExtractor/index.tsx

```diff
@@ -55,7 +55,6 @@ const bylineExtractor = (blocks: object[]) => {
     originCode,
     locator,
     resolution: 160,
-    isPng: true,
```

Comment:
Is this no longer required?

Reply:
Heya, by the looks of it, it's no longer used.